sway/input/seatop_resize_floating.c

```diff
@@ -75,16 +75,24 @@ static void handle_pointer_motion(struct sway_seat *seat, uint32_t time_msec,
 	int min_width, max_width, min_height, max_height;
 	floating_calculate_constraints(&min_width, &max_width,
 			&min_height, &max_height);
-	width = fmax(fmax(min_width + border_width, fmin(width, max_width - border_width)), 1);
-	height = fmax(fmax(min_height + border_height, fmin(height, max_height - border_height)), 1);
+	double bounded_width = fmax(min_width + border_width,
+								fmin(width, max_width - border_width));
```

Comment:
Suggestion: maybe this would be even more readable:

```c
width = fmin(width, max_width - border_width);
width = fmax(width, min_width + border_width);
width = fmax(width, 1);
```